util/gh-pages/script.js

```diff
@@ -258,6 +258,27 @@
                 return true;
             }
 
+            $scope.copyToClipboard = function (lint) {
+                const clipboard = document.getElementById("clipboard-" + lint.id);
+                if (clipboard) {
+                    let resetClipboardTimeout = null;
+                    let resetClipboardIcon = clipboard.innerHTML;
+
+                    function resetClipboard() {
+                        resetClipboardTimeout = null;
+                        clipboard.innerHTML = resetClipboardIcon;
+                    }
+
+                    navigator.clipboard.writeText("clippy::" + lint.id);
+
+                    clipboard.textContent = "✓";
```

Comment:
Oops, the `innerHTML` was left over from testing. Also, setting `textContent` to "&#10003;" doesn't replace it with the character for some reason. It might just be something it does at the parsing stage?

Reply:
The text content is mean to escape these AFAIK, for the code you have to use innerHTML again